kafka/consumer_test.go

```diff
@@ -65,6 +64,20 @@ func TestConsumerAPIs(t *testing.T) {
 		t.Errorf("Unsubscribe failed: %s", err)
 	}
 
+	topic := "gotest"
+	stored, err := c.OffsetsStore([]TopicPartition{{Topic: &topic, Partition: 0, Offset: 1}})
+	if err != nil && err.(Error).Code() != ErrUnknownPartition {
+		t.Errorf("OffsetsStore() failed: %s", err)
+		toppar := stored[0]
+		if toppar.Error.(Error).Code() == ErrUnknownPartition {
+			t.Errorf("OffsetsStore() TopicPartition error: %s", toppar.Error)
+		}
+	}
+	stored, err = c.OffsetsStore(nil)
```

Comment:
`nil` is a valid value for a slice, in fact it is the default value. I wanted to make sure it is explicitly handled. In the fixup I changed that to an uninitialized (default) slice.